airgun/views/virtwho_configure.py

```diff
@@ -0,0 +1,153 @@
+    class FilterUnlimitedForm(View):
+        pass
+
+    @filtering_content.register('Whitelist')
+    class FilterWhitelistForm(View):
+        filter_hosts = TextInput(
+            id='foreman_virt_who_configure_config_whitelist'
+        )
+        filter_host_parents = TextInput(
+            id='foreman_virt_who_configure_config_filter_host_parents'
+        )
+
+    @filtering_content.register('Blacklist')
+    class FilterBlacklistForm(View):
+        exclude_hosts = TextInput(
+            id='foreman_virt_who_configure_config_blacklist'
+        )
+        exclude_host_parents = TextInput(
+            id='foreman_virt_who_configure_config_exclude_host_parents'
+        )
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+                breadcrumb_loaded
+                and self.breadcrumb.locations[0] == 'Virt-who configurations'
+                and self.breadcrumb.read() == 'Create Config'
+        )
+
+
+class VirtwhoConfigureEditView(VirtwhoConfigureCreateView):
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+            breadcrumb_loaded
+            and self.breadcrumb.locations[0] == 'Virt-who Configurations'
+            and self.breadcrumb.read().startswith('Edit ')
+        )
+
+
+class VirtwhoConfigureDetailsView(BaseLoggedInView):
+    breadcrumb = BreadCrumb()
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+                breadcrumb_loaded
+                and self.breadcrumb.locations[0] == 'Configurations'
+                and self.breadcrumb.read() != 'Create Config'
+        )
+
+    edit = Text("//a[text()='Edit']")
+    delete = Text("//a[text()='Delete']")
+
+    @View.nested
+    class overview(SatTab):
+        TAB_NAME = 'Overview'
+        status = Text('.//span[contains(@class,"virt-who-config-report-status")]')
+        hypervisor_type = Text('.//span[contains(@class,"config-hypervisor_type")]')
+        hypervisor_server = Text('.//span[contains(@class,"config-hypervisor_server")]')
+        hypervisor_username = Text('.//span[contains(@class,"config-hypervisor_username")]')
+        interval = Text('.//span[contains(@class,"config-interval")]')
+        satellite_url = Text('.//span[contains(@class,"config-satellite_url")]')
+        hypervisor_id = Text('.//span[contains(@class,"config-hypervisor_id")]')
+        filtering = Text('.//span[contains(@class,"config-listing_mode")]')
+        debug = Text('.//span[contains(@class,"config-debug")]')
+
+    @View.nested
+    class deploy(SatTab):
+        TAB_NAME = 'Deploy'
+        command = Text("//pre[@id='config_command']")
+        script = Text("//pre[@id='config_script']")
```

Comment:
Because there is no id defined for these overview elements:
```
          <div class="row"><div class="col-md-2"><strong>Hypervisor Type</strong></div><div class="config-status"><span class="config-hypervisor_type">VMware vSphere / vCenter (esx)</span></div></div>
          <div class="row"><div class="col-md-2"><strong>Hypervisor Server</strong></div><div class="config-status"><span class="config-hypervisor_server">10.xxx.xxx.xxx</span></div></div>
```

Reply:
Ah, I see your point.